### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Newquery_LabelsorInstructions: Visual label is not provided for "Select database" dropdown.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio-mysql#178]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #GH_MySQLextensionforAzureDataStudio_Linux_Mar23; #MySQL extension for Azure Data Studio; #WCAG3.3.2; #Linux; #External; #External:AzureDataStudio; #FirstParty:22675;

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to "File" and hit ENTER key. TAB to "New Query" and hit ENTER key.
4. TAB to "Select database" and observe that Visual label is not provided for "Select database" dropdown.

### Actual Result:​
Visual label is not provided for "Select database" dropdown.

### Expected Result:​
Appropriate visual label should be provided for "Select database" dropdown.

### User Impact:
Screen reader users will not the purpose of the dropdown and will not get information about it.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:
![Uploading Github label.png…]()

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by PoojaNamde** [comment]
@nitishshamra25

As per the external bug process we have to log external bugs to MySQL extension. Please reactivate all the bugs and share all the details like VSO, area path details to log an external bug

**Comment by PoojaNamde** [comment]
Logged External bug: https://github.com/microsoft/azuredatastudio/issues/22675

As per the [External Bug Process - First Party Dependencies](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fGM4NTQxMGIxZGFhYjQwMTZmNTIzMDhkYWM5NDgyMWQwfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwNDM2MDg4Njg1MjIxNDZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkyT0RjMk5EQTROVFkzT1RzeE5qWTROelkwTURnMU5qYzVPekU1T2pjelpqUTNOV00zTFRnNU5Ea3RORFprT1MxaFlURTVMVGN3WlRobU9EWm1ZVGszT0Y4M1lqbG1ZVFV5TmkweVptRXhMVFF5WmpRdFltVXhOeTFsTXpRME5EbGpabVkyTlRSQWRXNXhMbWRpYkM1emNHRmpaWE09fDA5YTBhOTU1NzM0MDQ5NTM1OTJiMDhkYWM5NDgyMWQwfGZlNmZiOTdlMzBhNTRjNzJhNTRhNTU0Y2RjNDg0NzM3&sdata=ZVRCdGg0WldPVlRVQ2RveFFNV1FJUEc0bHc1a2QvWTExbkVLS3V5emQzYz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-ramakatare%40microsoft.com) assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:05-18-23

Verified this issue in the latest Azure data studio- Insider (Version: 1.45.0 Insider).
Issue is fixed, hence closing the bug.
